pyaedt/icepak.py

```diff
@@ -155,7 +155,7 @@ def assign_openings(self, air_faces):
         bound = BoundaryObject(self, boundary_name, props, 'Opening')
         if bound.create():
             self.boundaries.append(bound)
-            self._messenger.add_info_message("Opening Assigned")
+            self._messenger.add_info_message("Opening assigned.")
```

Comment:
@PipKat this change has affected the docstring example output on line 142. You can see the docstring testing failed on the CI check. Can you please update line 142?